scripts/run_registry.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash  
```

Comment:
This is a script that handles the local registry instance with a variant of `docker run registry:2`. Challenges here were:

* Need to compute a valid IP of the host that is visible from the qemu VM in "user mode" network setup.
* Need to push to an insecure registry (I could not figure out how work around that, a self signed certificate would still be insecure unless the host is configured to trust it)